Model/Service/Order.php

```diff
@@ -0,0 +1,176 @@
+<?php
+/**
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the MIT License
+ * It is available through the world-wide-web at this URL:
+ * https://tldrlegal.com/license/mit-license
+ * If you are unable to obtain it through the world-wide-web, please send an email
+ * to support@buckaroo.nl so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade this module to newer
+ * versions in the future. If you wish to customize this module for your
+ * needs please contact support@buckaroo.nl for more information.
+ *
+ * @copyright Copyright (c) Buckaroo B.V.
+ * @license   https://tldrlegal.com/license/mit-license
+ */
+namespace Buckaroo\Magento2\Model\Service;
```

Comment:
service classes, in general, should be placed under Service not Model/Service.

Reply:
it's not true
check magento sources
vendor/magento/module-store/Model/Service
vendor/magento/module-sales/Model/Service
also,our module already had Model/Service too